libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/SkuInterceptor.kt

```diff
@@ -0,0 +1,23 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import android.content.Context
+import com.mapbox.services.android.navigation.v5.internal.accounts.MapboxNavigationAccounts
+import java.io.IOException
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class SkuInterceptor(private val context: Context) : Interceptor {
```

Comment:
This accepts a non-null `Context`, is there any possibility in which this could become `null` as part of client use case and causing an unexpected crash?

I've seen that we're forcing non-null `Context` in `Billing` and `MapboxNavigationAccounts` but we're checking that `Context` is not null from the call-site but not for `SkuInterceptor`.